homeassistant/components/modbus/binary_sensor.py

```diff
@@ -73,6 +74,11 @@ async def async_setup_slaves(
         # this ensures that idx = bit position of value in result
         # polling is done with the base class
         name = self._attr_name if self._attr_name else "modbus_sensor"
+
```

Comment:
The changes in the modbus integration does not seem related to the title, please either explain or remove.

Reply:
Should only contain my changes now